cmd/resources/resource_utils.go

```diff
@@ -175,3 +189,7 @@ func getFlagName(paramName string) string {
 	}
 	return flagName
 }
+
+func replaceBackticks(s string) string {
+	return strings.Replace(s, "`", "'", -1)
+}
```

Comment:
for some weird reason cobra [takes the param description and adds it as the type](https://github.com/spf13/pflag/blob/d5e0c0615acee7028e1e2740a11102313be88de1/flag.go#L573):
```
Params: []Param{
	{
		Name:        "type",
		In:          "path",
		Description: "The type of event to retrieve. Must be either `received` or `published`.",
		Type:        "string",
	},
}
```
![Screenshot 2024-05-28 at 1 20 09 PM](https://github.com/launchdarkly/ldcli/assets/55991524/4730eaaf-6d6f-4811-b0ec-95bc17a20573)

so as a hack I'm just replacing the backticks with `'`'s. This does update some existing commands but it shouldn't be a breaking change.